<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login_window</class>
 <widget class="QMainWindow" name="Login_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>326</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>205</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>999</width>
    <height>999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Logowanie</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(249, 198, 171, 255), stop:1 rgba(232, 109, 52, 255));
font: bold 20pt &quot;Routed Gothic&quot;;
border-radius: 35px;
}
#centralwidget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(249, 198, 171, 255), stop:1 rgba(232, 109, 52, 255));
}
#checkBox_remember{
font: bold 11pt &quot;Routed Gothic&quot;;
color:white
}
#button_login{
background-color: rgb(248, 222, 214);
border-radius: 20px;
font: bold 15pt &quot;Routed Gothic&quot;;
color: rgba(224,102,46,99)
}
#button_login:hover {
background-color: rgba(228, 78, 9, 255);
color: white
}
#button_register{
background-color: rgb(248, 222, 214);
border-radius: 15px;
font: bold 10pt &quot;Routed Gothic&quot;;
color: rgba(224,102,46,99)
}
#button_register:hover {
background-color: rgba(228, 78, 9, 255);
color: white
}
#button_forgot{
background-color: rgb(248, 222, 214);
border-radius: 15px;
font: bold 10pt &quot;Routed Gothic&quot;;
color: rgba(224,102,46,99)
}
#button_forgot:hover {
background-color: rgba(228, 78, 9, 255);
color: white
}
#label_Haslo{
border-radius: 35px;
font: bold 14pt &quot;Routed Gothic&quot;;
color:white
}
#label_Logowanie{
border-radius: 35px;
font: bold 20pt &quot;Routed Gothic&quot;;
color:white
}
#label_Nazwa{
border-radius: 35px;
font: bold 14pt &quot;Routed Gothic&quot;;
color:white
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Logowanie">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Routed Gothic</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LOGOWANIE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Nazwa">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Routed Gothic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nazwa użytkownika:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Haslo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Routed Gothic</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Hasło:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_login">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Routed Gothic</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Zaloguj</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_remember">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Zapamiętaj mnie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_forgot">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Routed Gothic</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nie pamiętam hasła</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_register">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Routed Gothic</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Zarejstruj się</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
